calcom/cal.com
# Fix - add team members to emails

## What does this PR do?



This PR fixes a bug where team members were not included in the email and calendar bodies. This PR allows sending team emails to the member's set language.

![image](https://user-images.githubusercontent.com/65426560/219926503-d3f6a90b-530c-444d-ae0b-61131a71203e.png)

![image](https://user-images.githubusercontent.com/65426560/219927182-41136bc0-8799-4581-8ba2-ef8940376398.png)


Fixes #6456



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Create both a collective and a round-robin event
    - For the round-robin event, set a fixed host
    - For one of the team members to a different language
- For both event types do the following actions and ensure that the emails contain team members
    - Request booking
    - Create booking
    - Reschedule
    - Delete 

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
